pkgs/development/libraries/qt-5/5.8/qtwebengine/default.nix

```diff
@@ -41,18 +43,18 @@ qtSubmodule {
       -e "s,QLibraryInfo::location(QLibraryInfo::TranslationsPath),QLatin1String(\"$out/translations\"),g" \
       -e "s,QLibraryInfo::location(QLibraryInfo::LibraryExecutablesPath),QLatin1String(\"$out/libexec\"),g" \
       src/core/web_engine_library_info.cpp
-
+ '' + optionalString (!stdenv.isDarwin) ''
     sed -i -e '/lib_loader.*Load/s!"\(libudev\.so\)!"${systemd.lib}/lib/\1!' \
       src/3rdparty/chromium/device/udev_linux/udev?_loader.cc
 
     sed -i -e '/libpci_loader.*Load/s!"\(libpci\.so\)!"${pciutils}/lib/\1!' \
       src/3rdparty/chromium/gpu/config/gpu_info_collector_linux.cc
   '';
 
-  qmakeFlags = lib.optional enableProprietaryCodecs "WEBENGINE_CONFIG+=use_proprietary_codecs";
+  qmakeFlags = optional enableProprietaryCodecs "WEBENGINE_CONFIG+=use_proprietary_codecs";
 
   propagatedBuildInputs = [
-    dbus zlib minizip alsaLib snappy nss protobuf jsoncpp libevent
+    #dbus zlib minizip snappy nss protobuf jsoncpp libevent
```

Comment:
thx, i'll have to give qtwebengine and qtwebkit a special look by rewriting the sed-stuff i mentioned above. The main reason is that because of the the sed-stuff OpenGL and CoreFoundation cannot be propagated correctly.